### DISCUSSION TITLE
Touchpad Not Working on Lenovo IdeaPad Flex 5 14ARE05 with Brunch/Chrome OS: Kernel Parameter Help Needed

### DISCUSSION THREAD

**Discussion by SvenThe** [sebanc/brunch#2095] [Q&A]
Hello Community,

I'm facing an issue with my Lenovo IdeaPad Flex 5 14ARE05 and I hope someone can help. I've set up a dual-boot system with Windows 11 and Chrome OS (via Brunch). Unfortunately, my touchpad isn't working under Chrome OS.

I suspect I need to modify the kernel parameters to solve this problem. After some research, I've come across the following kernel parameters that might help:

i8042.nopnp=1 pci=nocrs
i8042.noloop
i8042.nomux
i8042.nopnp
i8042.reset
module_blacklist=i2c_hid,i8042
module_blacklist=hid_multitouch
psmouse.proto=imps
psmouse.elantech_smbus=0
psmouse.synaptics_intertouch=0
module_blacklist=elan_i2c

1. Does anyone have experience with this specific laptop model and Chrome OS?
2. Are these kernel parameters appropriate for my issue, or are there better alternatives?
3. What's the best way to incorporate these parameters into my Brunch installation? I've tried editing the brunch.ini file, but I didn't found it.
4. Are there any other tips or tricks I should try to get my touchpad working?

I appreciate any help you can provide. If you need any additional information, please let me know.

Thank you in advance!

**Comment by csp5me** [comment]
I have the same model as yours. Bought in 2020 with AMD 4500U cpu. I don't know if Lenovo uses different touchpads for the same model. 

But Brunch is working just fine. First I used kernel 6.1. Now 6.6. Both versions work.
No driver specific brunch patches needed, other than crosh sudo, pwa updates, the power button and suspend. 
Only tablet mode or screen rotation does not work. For me not important.
